parts/kuberneteskubelet.service

```diff
@@ -41,7 +39,9 @@ ExecStart=/usr/bin/docker run \
         --cluster-dns=${KUBELET_CLUSTER_DNS} \
         --cluster-domain=cluster.local \
         --register-schedulable=${KUBELET_REGISTER_SCHEDULABLE} \
-        --node-labels="${KUBELET_NODE_LABELS},beta.kubernetes.io/instance-type=${KUBELET_VMSIZE},failure-domain.beta.kubernetes.io/zone=${KUBELET_FD}" \
```

Comment:
@anhowe @dmitsh Makes sense to remove this for now as to not hit ARM limits, but we need to track that these labels are now missing and have a plan to put them back. Teams use them in designing for high-availability and for maintenance.